packages/ecommerce-app-base/package.json

```diff
@@ -35,7 +35,7 @@
   },
   "dependencies": {
     "@contentful/app-sdk": "^4.0.0",
-    "@contentful/f36-components": "^4.0.0",
```

Comment:
Shall we not go to the latest?

Reply:
The goal is to maximize the version range. 4.0.42 is the first version that is working with the app-base.